packages/front-end/components/Markdown/CustomMarkdown.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Handlebars from "handlebars";
+import Markdown from "./Markdown";
+
+interface Props {
+  markdown?: string;
+  handlebarsVariables: Record<string, any>;
+}
+
+const CustomMarkdown: React.FC<Props> = ({ markdown, handlebarsVariables }) => {
```

Comment:
Add a standard set of variables that are always available (user, orgName).  Only pass in additional ones as needed